polaris.shopify.com/content/contributing/documention.md

```diff
@@ -5,64 +5,161 @@ keywords:
   - how to contribute to the style guide
   - documenting in polaris
   - documentation guidelines
+quickGuides:
+  - title: Making copy edits (grammatical errors, typos, etc.)
+    queryParam: making-copy-edits
+    rows:
+      - openSource:
+        question: Are open source contributions accepted?
+        answer: Yes
+      - who:
+        question: Who should I involve?
+        answer: Subject matter experts, impacted teams, and or those with the most context on the Polaris team.
+      - when:
+        question: When should I contribute?
+        answer: As needed
+      - where:
+        question: Where do I contribute?
+        answer: Copy edits can be contributed in the <a href="https://github.com/Shopify/polaris">Shopify/polaris</a> GitHub repo. Copy edits should also be contributed in Figma if relevant.
+      - relatedIssues:
+        question: Where should I search for related issues or topics?
+        answer: Search for related issues and pull requests in the <a href="https://github.com/Shopify/polaris">Shopify/polaris</a> GitHub repo.
+      - what:
+        question: What resources will my contribution impact?
+        answer: Along with related content sections on this documentation site, your contribution may impact the Figma UI Kit and component examples.
+      - how:
+        question: How can I get started?
+        answer: <a href="/contributing/shipping-your-contribution">Open a pull request</a> in the <a href="https://github.com/Shopify/polaris">Shopify/polaris</a> GitHub repo
```

Comment:
This approach to content is complex and should be avoided. I think we can create a much simpler initial solution that is just a HTML page. I am unsure what value the `QuickStartGuide.tsx` component is giving.